src/main/java/chronologer/task/Event.java

```diff
@@ -31,6 +31,25 @@ public Event(String description, LocalDateTime startDate, LocalDateTime endDate)
         this.type = EVENT;
     }
 
+    /**
+     * creates a new Event task.
+     *
+     * @param description description of task
```

Comment:
Please do update the commenting style for javadocs capitalization etc. to standardize with the other javadocs.